share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -160,6 +160,15 @@ macro(rocm_create_package)
     if(ROCM_VERSION_NUM)
         set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION}.${ROCM_VERSION_NUM}")
     endif()
+    # Set the License to rpm header
+    if(PARSE_LICENSE)
+        set( CPACK_RPM_PACKAGE_LICENSE "${PARSE_LICENSE}" )
+    endif()
+    # Copy the license file to /opt/rocm/share/doc/<component_name>
+    if(PARSE_LICENSE_PATH)
+        set(CPACK_RESOURCE_FILE_LICENSE "${PARSE_LICENSE_PATH}")
+        install(FILES ${CPACK_RESOURCE_FILE_LICENSE} DESTINATION share/doc/${PARSE_COMPONENT})
```

Comment:
`PARSE_COMPONENT` will not be defined.

Reply:
The install statement is also redundant (see line 344) and not entirely correct, as it installs the license only to the `runtime` component, while header only libraries need this file installed to the `devel` component.

Reply:
For reference, the desired location is:

share/doc/XXX

where XXX is either the cmake project name OR the package name.

E.g. share/doc/rocrand for "rocRAND"

Reply:
@lawruble13  thanks for the comments. PARSE_COMPONENT was a copy paste mistake. Correcting it to CPACK_PACKAGE_NAME. 
If devel package is depended on main package we need the license file only in 1 package. Like Jeremy pointed out rocrand and rocrand-dev package licence will be in share/doc/rocRAND

Reply:
@nunnikri For header only libraries, there is no runtime package (as there would be no contents to that package), so the license file must be installed to the devel package. For all other libraries, the runtime component doesn't depend on the devel component, so the runtime component must be the component which has the license file.

Reply:
The installation is already handled on the aforementioned lines, simply removing the install statement on line 170 should work.

Reply:
Thanks Liam. The changes are updated as per your comments